src/main/java/com/google/api/codegen/transformer/StandardImportSectionTransformer.java

```diff
@@ -32,7 +36,21 @@ public ImportSectionView generateImportSection(TransformationContext context, St
   @Override
   public ImportSectionView generateImportSection(
       MethodContext context, Iterable<InitCodeNode> specItemNodes) {
-    return generateImportSection(context.getTypeTable().getImports(), null);
+    boolean importIOUtility =
```

Comment:
Just a nit: in these files, you might communicate your intent more clearly if you s/`importIOUtility`/`needIOUtility`/

Also cosmetic: consider s/`IOUtility`/`FileUtility`/g to avoid the 3 caps in a row (but  totally up to you)